spec/components/topic_query_spec.rb

```diff
@@ -861,6 +861,59 @@ class ::TopicQuery
     end
   end
 
+  context 'list_unseen' do
```

Comment:
```suggestion
  context '#list_unseen' do
```

Generally, our codebase follows https://www.betterspecs.org/#describe in describing methods which we're testing.